lib/src/app/app.dart

```diff
@@ -379,10 +379,11 @@ class AppClient {
   }
 
   /// Get a list of [Fragment]s in an [Organization]
-  Future<List<Fragment>> listFragments(String organizationId, bool showPublic) async {
+  Future<List<Fragment>> listFragments(String organizationId, {bool? showPublic}, {FragmentVisibility? visibility}) async {
```

Comment:
changed this to be optional given that it's an optional field in api - let me know if that seems wrong